### PR TITLE
addresses + fixes

### PR DESCRIPTION
**Pull request by absoIute**

### CODE REVIEW

File: bindings/2.206/GeometryDash.bro [hunk 1]
```diff
@@ -6400,7 +6400,7 @@ class GJBaseGameLayer : cocos2d::CCLayer, TriggerEffectDelegate {
 	TodoReturn prepareTransformParent(bool);
 	TodoReturn preResumeGame();
 	TodoReturn preUpdateVisibility(float);
-	TodoReturn processActivatedAudioTriggers(float);
+	TodoReturn processActivatedAudioTriggers(float) = win 0x232a70;
```
Comment (Fleeym) [inline bindings/2.206/GeometryDash.bro:6403]: mr Absolute please make this void thank you

File: bindings/2.206/GeometryDash.bro [hunk 2]
```diff
@@ -12259,25 +12259,9 @@ class PlayLayer : GJBaseGameLayer, CCCircleWaveDelegate, CurrencyRewardDelegate,
 	void delayedFullReset();
 	void delayedResetLevel() = win 0x395700;
 	void fullReset() = win 0x395600;
-	float getCurrentPercent() = win inline {
```
Comment (Prevter) [inline bindings/2.206/GeometryDash.bro:12262]: getCurrentPercent() is inlined on imac, so would be great if you didn't remove the definition entirely

Comment (absoIute) [reply bindings/2.206/GeometryDash.bro:12262]: mb, is that for intel macs, ASi or both?

Comment (Prevter) [reply bindings/2.206/GeometryDash.bro:12262]: not sure about m1 macs, but i was looking for intel macs bindings an hour ago which had this function inlined

Comment (absoIute) [reply bindings/2.206/GeometryDash.bro:12262]: should be sorted now

**Comment by Prevter** [comment]
can you also please fix member test for windows after changing `m_unk1f8` to `m_currentProgress`? thanks

**Comment by Fleeym** [comment]
shit I already merged, will fix myself

**Pull request merged**